dotcom-rendering/src/components/FrontSectionTitle.tsx

```diff
@@ -16,27 +23,65 @@ const titleStyle = css`
 	}
 `;
 
+const advertisingPartnerDottedBorder = css`
+	border-bottom: 1px dotted ${sourcePalette.neutral[86]};
+	margin-top: 0.375rem;
+`;
+
+const brandingAdvertisingPartnerStyle = css`
+	margin: 4px 0 20px;
+	padding: 4px;
+	border: 1px solid ${palette('--branding-border')};
+	width: fit-content;
+
+	${from.desktop} {
+		padding: 8px;
+		width: 220px;
+	}
+	${from.leftCol} {
+		padding: 4px;
+		width: fit-content;
+	}
+	${from.wide} {
+		padding: 8px;
+		width: auto;
+	}
+`;
+
+const labelAdvertisingPartnerStyles = css`
+	margin-top: 0;
+	border-top: 0;
+`;
+
 const labelStyles = css`
 	${textSans12};
 	line-height: 1rem;
-	color: ${palette.neutral[46]};
+	color: ${sourcePalette.neutral[46]};
 	font-weight: bold;
 	margin-top: 0.375rem;
 	padding-right: 0.625rem;
 	padding-bottom: 0.625rem;
 	text-align: left;
-	border-top: 1px dotted ${palette.neutral[86]};
+	border-top: 1px dotted ${sourcePalette.neutral[86]};
 `;
 
 const aboutThisLinkStyles = css`
 	${textSans12};
 	line-height: 11px;
-	color: ${palette.neutral[46]};
+	color: ${sourcePalette.neutral[46]};
 	font-weight: normal;
 	text-decoration: none;
 `;
 
-export const FrontSectionTitle = ({ title, collectionBranding }: Props) => {
+const aboutThisLinkAdvertisingPartnerStyles = css`
+	color: ${sourcePalette.news[400]};
```

Comment:
I fear this may not offer enough contrast in dark mode. Would it be possible to use a scheme-aware palette token?

Reply:
I understand that but we were given 1 colour for the light mode and dark mode #B7271B which I didn't even find in the source colours so I had to pick the closest one to it. Is the schema-aware palette token like `${palette('--branding-border')}`?

Reply:
As we do not support dark mode on Web yet and this targets only fronts, I do not think it’s a blocker for this piece of work :)